core/Plugin/Dimension/ActionDimension.php

```diff
@@ -191,6 +191,23 @@ public function onNewAction(Request $request, Visitor $visitor, Action $action)
     }
 
     /**
+     * Returns `true` if the column for this dimension is a reference to the `log_action` table (ie, an "idaction column"),
+     * `false` if otherwise.
+     *
+     * @return bool
+     */
+    public function isActionReference()
```

Comment:
If we put this method in `ActionDimension` it should be marked as `@ignore` or better `@internal` and maybe even marked as `final` as developers should not be able to change that behaviour. Or maybe it is possible to put it in a different class? Not sure what it could be called though.